### ISSUE TITLE
Copy-paste error in configure.in?

### ISSUE THREAD

**Issue by jhwillett** [citusdata/citus#4833]
On master at `697bbbd `, https://github.com/citusdata/citus/blob/master/configure.in#L240 reads:
```
  Use --without-lz4 to disable zlib support.
```
Should this be:
```
  Use --without-lz4 to disable lz4 support.
```
?

Apologies for not simply offering a PR with this diff, I have too many action items in flight concurrently and I doubt I could follow up properly.

**Comment by jhwillett** [comment]
Never mind, I was able to clear my head for a few minutes: https://github.com/citusdata/citus/pull/4834.

**Comment by marcocitus** [comment]
Thanks! Fixed by #4834 and #4835.

**Comment by jhwillett** [comment]
Much obliged!
